src/org/ods/component/HelmDeploymentStrategy.groovy

```diff
@@ -95,8 +95,10 @@ class HelmDeploymentStrategy extends AbstractDeploymentStrategy {
         logger.info "Rolling out ${context.componentId} with HELM, selector: ${options.selector}"
         helmUpgrade(context.targetProject)
 
+        def deploymentKinds = DEPLOYMENT_KINDS << 'StatefulSet'
```

Comment:
Why not actually creating OpenShiftService.STATEFUL_SET_KIND and adding it to the DEPLOYMENT_KINDS list? Perhaps this list could be renamed to WORKLOAD_KINDS or something similar.